RetroShare.pro

```diff
@@ -17,7 +17,9 @@
 ################################################################################
 
 !include("retroshare.pri"): error("Could not include file retroshare.pri")
-
+#atai add this one
+CONFIG += qt gui uic qrc resources uitools debug
```

Comment:
I dont think this is needed. libretroshare that is below this directory does not need qt. All this should be in retroshare-gui/src/retroshare-gui.pro, and it's probably already there.